### DISCUSSION TITLE
Potential breakout: Bring your own dataset (BYOD)

### DISCUSSION THREAD

**Discussion by ml-evs** [MADICES/MADICES-2022#23] [Ideas]
Another potential breakout we could use, if other discussions have been exhausted (or just if there is enough interest). This one might be particularly valuable for scientists in attendance.

I guess the vague idea would be that people give 1-2 slides on a dataset they have published, want to publish, or are currently creating, and they can ask for feedback on how to make it FAIR/interoperable. The more awkward the dataset, the better!

If the dataset has been published already, other room attendees can try to download and explore and give feedback. Otherwise, a data expert can provide some useful insight into how they would approach the problem, and we can see how feasible their approach would be for the individual researcher.

**Comment by kjappelbaum** [comment]
"promised" this already on Twitter https://twitter.com/kmjablonka/status/1479768810714456064?s=20 🙊

**Reply by ml-evs** [reply]
Let's upgrade this from a potential breakout to a real one then :)
